index.html

```diff
@@ -5,9 +5,357 @@
+                            Login
+                        </button>
+                    </div>
+                </div>
+                <div class="hero-content-img">
+                    <img src="img/Main-TV-frame.png" alt="zdjęcię przedstawiające telewizor">
+                    <img class="hero-img-phone" src="img/Main-phone-frame.png" alt="zdjęcię przedstawiające telefon">
+                    <div class="hero-absolute-box flex-center">
+                        <img class="hero-live" src="img/live.png" alt="na żywo">
+                    </div>
+                </div>
+            </div>
+            <ul class="space-between hero-image-list">
+                <li class="hero-platforms">
+                    <img src="img/tv.png" alt="napis apple-tv">
+                </li>
+                <li class="hero-platforms">
+                    <img src="img/android.png" alt="napis android-tv">
+                </li>
+                <li class="hero-platforms">
+                    <img src="img/appleexport.png" alt="napis apple-watch">
+                </li>
+                <li class="hero-platforms">
+                    <img src="img/iphone.png" alt="napis iphone">
+                </li>
+                <li class="hero-platforms">
+                    <img src="img/androidicon.png" alt="napis android">
+                </li>
+            </ul>
+        </div>
+    </section>
+    <section class="flex-center">
+        <div class="section-layout">          
+            <h2 class="tile-section-title dark text-center title-text">
+                All-in-one platform
+            </h2>
+            <p class="subtxt text-center dark paragraph-text">
+                You take care of the video quality and we take care of <br> everything else.
+            </p>
+            <div class="tile-section-cta centrally-justified">
+                <button class="tile-left-button blue-background button">
+                    Start Free Trial
+                </button>
+                <button class="tile-right-button dark button">
+                    Login
+                </button>
+            </div>
+            <div class="tile-list-box">
+                <ul class="tile-list space-between">
+                    <li class="tile-list-item">
+                        <div class="centrally-aligned">
+                            <img src="img/upload.png" alt="ikona wrzucania pliku">
+                            <h3 class="tile-list-subtitle dark subtitle-text">
+                                Upload & Organize
+                            </h3>
+                        </div>
+                        <p class="tile-list-txt dark paragraph-text">
+                            Upload in bulk, organze content in categories, add custom filters & upload extras.
+                        </p>
+                    </li>
+                    <li class="tile-list-item">
+                        <div class="centrally-aligned">
```

Comment:
a może po prostu tile-title ? czego by tu nie pokazywać to będzie zawsze tytuł kafelka